### DISCUSSION TITLE
ZEPETO 월드에서 지원하는 유니티 패키지 목록에 포즈추정을 위한 바라쿠다 패키지를 추가해 주실 수 있나요?

### DISCUSSION THREAD

**Discussion by Kim-JeongUng** [naverz/zepeto-studio-kor#871] [5. Q&A]
https://github.com/Kim-JeongUng/ZEPETO-Unity-Barracuda-MotionCapture/

카메라 / 비디오를 통한 모션캡쳐를 제페토 캐릭터에 리깅 되도록 샘플을 제작했습니다.

![ezgif com-gif-maker (1)](https://user-images.githubusercontent.com/82865325/168556151-e79b6850-ba0b-4e8b-b3c6-41e43b083b31.gif) 
다만 [ZEPETO 월드에서 지원하는 유니티 패키지 목록](https://studio.zepeto.me/kr/guides/unity_package_list)에 바라쿠다 패키지가 없어 빌드시엔 동작되지 않습니다.

혹시 내부적으로 검토 후 패키지가 추가될 수 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 내부적으로 검토해 보겠습니다. 
감사합니다.
